benchmark/tool/C/README.md

```diff
@@ -35,7 +36,17 @@ To compile and run this demo in an Android environment, please follow the follow
 
     make
     ```
-3. Run the demo program by logging into the Android environment via adb and specifying the PaddlePaddle model from the command line as follows:
-```
-./inference --merged_model ./model/mobilenet.paddle --input_size 150528
-```
+
+3. Prepare a merged model:   
+Models config(.py) (eg: [Mobilenet](https://github.com/PaddlePaddle/Mobile/blob/develop/models/mobilenet.py)) contain only the structure of our models. A developer can choose [model config here](https://github.com/PaddlePaddle/Mobile/tree/develop/models) to train their custom models. PaddlePaddle documentation has [several tutorials](https://github.com/PaddlePaddle/models) for building and training models. The model parameter file(.tar.gz) will be generated during the training process. There we need to merge the configuration file(.py) and the parameter file(.tar.gz) into a file. Please refer to the [details.](https://github.com/PaddlePaddle/Mobile/tree/develop/tools/merge_config_parameters)
+
+4. Run the demo program by logging into the Android environment via [adb](https://developer.android.google.cn/studio/command-line/adb.html?hl=zh-cn#Enabling) and specifying the PaddlePaddle model from the command line as follows:
+	```
+	adb push inference libpaddle_capi_shared.so mobilenet.paddle /sdcard/test_mobilenet
```

Comment:
It seems this example does not link `libpaddle_capi_shared.so` but `libpaddle_capi_layers.a` and `libpaddle_capi_engine.a`. In this case, do not need to copy the library to the device.

Reply:
I see.